web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -31,6 +31,9 @@ import { ActivatedRoute } from '@angular/router';
  * project header. Responsible for coordinating page-level URL states with
  * various services.
  */
+
+const LAYER_ID_NEW = ':new';
```

Comment:
D.R.Y. - please use `RouterService.LAYER_ID_NEW` instead of redefining this constant here.

Reply:
Done :)